app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardView.kt

```diff
@@ -29,23 +29,26 @@ class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEd
     var isTranslation: Boolean = false
     var sourceSummary: RbPageSummary? = null
     var targetSummary: RbPageSummary? = null
-    var addedDescription: String? = null
 
     init {
-        inflate(getContext(), R.layout.fragment_suggested_edits_add_descriptions_item, this)
+        inflate(getContext(), R.layout.view_suggested_edit_card, this)
```

Comment:
You may need to update the import from the `
import kotlinx.android.synthetic.main.fragment_suggested_edits_add_descriptions_item.view.*` to the `
import kotlinx.android.synthetic.main.view_suggested_edit_card.view.*`

Reply:
Done